metal/eip_controlplane_reconciliation.go

```diff
@@ -69,138 +75,157 @@ func newControlPlaneEndpointManager(k8sclient kubernetes.Interface, stop <-chan
 			}},
 		eipTag:        eipTag,
 		projectID:     projectID,
-		instances:     i,
 		ipResSvr:      ipResSvr,
 		deviceIPSrv:   deviceIPSrv,
 		apiServerPort: apiServerPort,
 		k8sclient:     k8sclient,
-		stop:          stop,
+		useHostIP:     useHostIP,
 	}
-	klog.V(2).Info("controlPlaneEndpointManager.init(): enabling BGP on project")
+
```

Comment:
At later points, when processing each node, we check if `m.eipTag == ""` in order to know to do nothing. Should we not do it here as well? Why set this all up if `eipTag == ""`?

Reply:
As you suggested, I moved the validation for eipTag to avoid configuring all the reconciliation logic if it's not set.